src/app/globals.css

```
@@ -364,26 +364,74 @@ html:not(.dark) [class*="nav-container"] {
+
+.vengeanceUI-text{
+
+  font-size: 6rem;
+  font-weight: bold;
+  text-transform: uppercase;
+  color: #fff;
+  position: relative;
+  animation: melt 3s infinite ease-in-out;
+  background: linear-gradient(90deg, #ff6f61 , #ffbd44 , #ff6f61);
+  background-clip: text;
+  color:transparent
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)